### PR TITLE
Integrate file sync download to automatically import downloaded data into Redis 

### PR DESCRIPTION
**Pull request by jackwherry**
To test these changes,
* chase-car-dashboard must be on the `file_sync_integration` branch
* file_sync (now a  submodule of chase-car-dashboard) must be on the `database_import` branch

See https://github.com/badgerloop-software/chase-car-dashboard/pull/76

### CODE REVIEW

**Comment by MCLiii** [comment]
Looks good, once the chase car end is implemented and tested with call back I'll approve this

File: file_sync_down/main.py [hunk 1]
```diff
@@ -29,36 +32,43 @@ def download_zip_file(files_to_download: list):
...
         print("files to download:", files_to_download)
 
         if len(files_to_download) == 1:
-            download_single_file(files_to_download[0][0])
+            download_single_file(files_to_download[0][0], callback)
         elif len(files_to_download) > 1:
-            unzip_file(download_zip_file([f[0] for f in files_to_download]))
+            extracted_files = unzip_file(download_zip_file([f[0] for f in files_to_download]))
+            for file in extracted_files:
+                callback(file) # TODO check whether this gets the DATA or just a file handle
```
Comment (MCLiii) [inline file_sync_down/main.py]:
```suggestion
                with open(fs_config.sync_folder_location + "/" + file, "rb") as f:
                    callback(f.read())
```

**Review by MCLiii — CHANGES_REQUESTED** [review]
URL also needs to be changed to fs.bsr-dev.org

**Comment by jackwherry** [comment]
When this is ready to merge, we need to do it at the same time as https://github.com/badgerloop-software/chase-car-dashboard/pull/76

**Review by MCLiii — COMMENTED** [review]
Same url needs to be changed for file_sync_up

**Comment by jackwherry** [comment]
Done. Do you want to do full scale testing on this as soon as I’m done with the new OS stack on the Pi? (i.e., using the actual Pi and server, with the car just sitting there)

**Comment by MCLiii** [comment]
> Done. Do you want to do full scale testing on this as soon as I’m done with the new OS stack on the Pi? (i.e., using the actual Pi and server, with the car just sitting there)

I've done it with ethernet sim, seems to work. I am getting a url too long when there's too much files queued up, probably gonna increase the file size threashold for upload to 100kb and change zip downloads api to use post

**Pull request merged**